Exercises/strings/strings.js

```diff
@@ -0,0 +1,20 @@
+// STRINGS
+// 1. Assign string with unicode to a variable.
+let unicodeString = 'Example of unicode symbol is \u00B9';
```

Comment:
always `const` when you are not reassigning variable

Reply:
Resolved.